config/config.go

```diff
@@ -20,86 +20,51 @@ type Config struct {
 	Camera         map[string]string `json:"camera"`
 	MovementSensor map[string]string `json:"movement_sensor"`
 	ConfigParams   map[string]string `json:"config_params"`
-	DataDirectory  string            `json:"data_dir"`
-	MapRateSec     *int              `json:"map_rate_sec"`
-	Sensors        []string          `json:"sensors"`
-	DataRateMsec   *int              `json:"data_rate_msec"`
-
-	CloudStoryEnabled bool   `json:"cloud_story_enabled"`
-	ExistingMap       string `json:"existing_map"`
-	EnableMapping     *bool  `json:"enable_mapping"`
-	UseCloudSlam      *bool  `json:"use_cloud_slam"`
-	// NewConfigFlag is used to access the config changes from imu and cloud story work while providing backwards compatibility
-	// To be removed during JIRA Ticket: RSDK-4742 https://viam.atlassian.net/browse/RSDK-4742
-	NewConfigFlag bool `json:"new_config_flag"`
+
+	ExistingMap   string `json:"existing_map"`
+	EnableMapping *bool  `json:"enable_mapping"`
+	UseCloudSlam  *bool  `json:"use_cloud_slam"`
 }
 
 // OptionalConfigParams holds the optional config parameters of SLAM.
 type OptionalConfigParams struct {
-	LidarDataRateMsec int
-	ImuName           string
-	ImuDataRateMsec   int
-	MapRateSec        int
-	EnableMapping     bool
-	ExistingMap       string
+	LidarDataFrequencyHz int
```

Comment:
[note] switched to frequency hz throughout due to confusion between all the difference data rate secs during removal. Can revert this change if people would prefer DataRateMsec

Reply:
I wouldn't have included this change as part of this PR, since this PR to turn on cloud SLAM should be as low risk and easy to review as possible, to reduce the chance it introduces a bug. However, since you've already included it, I don't think you need to change it.